lib/tasks/db.js

```diff
@@ -69,22 +124,26 @@ module.exports = {
     descriptions: {
       'short': 'Deletes data from the database.',
       'long': [
-        'The command unseeds every existing seed.'
+        'The command tries unseeding every existing seed.'
       ]
     },
 
     task: function () {
       getMigrator('seeder', function (migrator) {
-        migrator.executed()
+        migrator.pending()
         .then(function (seeders) {
           if (seeders.length === 0) {
-            console.log('No executed seeders found.');
+            console.log('No seeders found.');
             process.exit(0);
           }
-        }).then(function () {
-          return migrator.down({to: 0});
+
+          return migrator.down({migrations: _.chain(seeders).pluck('file').reverse().value()});
```

Comment:
If you don't apply some seeds, I think unseeding all of them would not pose a problem. But I guess that depends on your 'down' implementation. From my viewpoint, a seed is DB information, and a delete or update would not cause errors.